doc/source/ray-core/ray-dashboard.rst

```diff
@@ -6,74 +6,306 @@ Ray provides a web-based dashboard for monitoring and debugging Ray applications
+This allows users to identify tasks and actors that may be consuming more resources than expected and optimize the performance of the application. 
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/node_cpu_by_comp.png
+    :align: center
+
+Per component CPU graph. 0.379 cores mean that it uses 40% of a single CPU core. If the name starts with ``ray::``, that's the name of the Ray processes. ``raylet``, ``agent``, ``dashboard``, ``gcs`` mean the system components.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/node_memory_by_comp.png
+    :align: center
+
+Per component memory graph. If the name starts with ``ray::``, that's the name of the Ray processes. ``raylet``, ``agent``, ``dashboard``, ``gcs`` mean the system components.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/cluster_page.png
+    :align: center
+
+Additionally, users can see a snapshot of hardware utilization from the :ref:`cluster page <dash-node-view>`, which provides an overview of resource usage across the entire Ray cluster.
 
-.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard/nodes.png
-   :align: center
+.. _dash-workflow-resource-utilization:
 
+See the Resource Utilization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ray requires users to specify the number of :ref:`resources <logical-resources>` their tasks and actors will use through arguments such as ``num_cpus``, ``num_gpus``, ``memory``, and ``resource``. 
+These values are used for scheduling, but may not always match the actual resource utilization (physical resource utilization).
+
+- You can see the logical and physical resource utilization over time from the :ref:`Metrics View <dash-metrics-view>`.
+- The snapshot of physical resource utilization (CPU, GPU, memory, disk, network) is also available from the :ref:`Cluster View <dash-node-view>`.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/logical_resource.png
+    :align: center
+
+The :ref:`logical resources <logical-resources>` usage.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/physical_resource.png
+    :align: center
+
+The physical resourcecs (hardware) usage. Ray provides CPU, GPU, Memory, GRAM, disk, and network usage of each hardware in a cluster.
+
+.. _dash-overview:
+
+Overview
+--------
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/new-dashboard-v2/dashboard-pics/overview-page.png
+    :align: center
+
+The overview page provides a high-level status of the Ray cluster.
+
+Overview Metrics
+~~~~~~~~~~~~~~~~
```

Comment:
It's a bit hard to differentiate the h2 and h3 headers.
Maybe just use bold body text for the 3 sub-titles?